nixos/lib/make-ext4-fs.nix

```diff
@@ -37,32 +37,28 @@ pkgs.stdenv.mkDerivation {
       ${populateImageCommands}
       )
 
-      # Add the closures of the top-level store objects.
-      storePaths=$(cat ${sdClosureInfo}/store-paths)
+      # Create nix/store before copying path
+      mkdir -p ./rootImage/nix/store
+
+      # Also include a manifest of the closures in a format suitable for nix-store --load-db
+      cp ${sdClosureInfo}/registration ./rootImage/nix-path-registration
+
+      echo "copying store paths for image..."
```

Comment:
```suggestion
      echo "copying store paths to image..."
```

Reply:
It's not really copying *to* image, image is created *from* these files.

Reply:
Updated message, should be more obvious now what it's doing.